cmd/dbmigrate/utils/migrator.go

```diff
@@ -460,11 +466,14 @@ func (m *migrationManager) MigrateDBDir(dbDir string) (summary string, err error
 	}()
 
 	m.Status = "detecting db type"
-	var ok bool
-	sourceDBType, ok = DetectDBType(dbName, sourceDir)
+	sourceDBType, ok := tmdb.BackendType(m.Migrator.SourceDBType), true
```

Comment:
```suggestion
	var ok bool
	sourceDBType, ok = tmdb.BackendType(m.Migrator.SourceDBType), true
```
Using `:=` here creates a new var which shadows the one above that would get used in the error.

Reply:
Nevermind. All good: https://go.dev/play/p/W_hgqMD6M9f?v=goprev
https://go.dev/ref/spec#Short_variable_declarations
It's not creating a new/shadow var because `sourceDBType` is defined earlier in the same block, and go handles it.